projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.coponent.ts

```diff
@@ -1,40 +1,17 @@
 import { Component } from '@angular/core';
+import { NavGroup } from '../../../../../../../go-lib/src/public_api';
 
```

Comment:
The file name here is misspelled - should be `badge-docs.component.ts` (missing 'm')